src/CDPVault.sol

```diff
@@ -757,7 +757,7 @@ abstract contract CDPVault is AccessControl, Pause, Permission, InterestRateMode
 
     /// @notice Returns the current spot price of the collateral token
     /// @return _ Current spot price of the collateral token [wad]
-    function spotPrice() public returns (uint256) {
+    function spotPrice() public view returns (uint256) {
```

Comment:
should be left as non view. otherwise interface might break going forward. because there are lots of price sources which are non-static.

Reply:
view required for compatibility of AuraVault with ERC-4626